index.js

```diff
@@ -0,0 +1,33 @@
+import Book from './modules/book-class.js';
+import * as navBarLinks from './modules/nav_bars.js';
+
```

Comment:
- [ ] Kindly make sure to add luxon to date and time as per the requirements.
- [ ] Also make sure your app works correctly before submitting it for the following review.

Reply:
@sja-thedude  Please about the luxon package, I spent more than 4 hours on it I don't get it and I don't know what to do about it. I installed the package but I can't use it, 'require' is not working for it likewise 'import', I copied the file my linters complaint about having more than one class, I was really frustrated about it that's why I didn't implement it in my code. I'll be grateful if I can have an insight about it so that I'll be able to proceed. Thanks